### PR TITLE
Expenses tab: Balance screen

### PR DESCRIPTION
**Pull request by mikejensen0**
Implements issue #19 

**IMPORTANT:** merge PR #37 before reviewing this one.

### CODE REVIEW

File: app/(tabs)/expenses/index.tsx
Comment (UnluckyBird) [inline app/(tabs)/expenses/index.tsx:1]: Isn't this screen added in the settle PR, should try to not have the same changes in several PRs

**Review by UnluckyBird — APPROVED** [review]
Balance tab looks good, not sure why settle was added, but if it works it's probably fine

File: helpers/calculateExpenses.tsx
[the remark this answers is not in the document]
Comment (mikejensen0) [reply helpers/calculateExpenses.tsx:1]: It is now fixed.

File: app/(tabs)/expenses/index.tsx
Comment (mikejensen0) [reply app/(tabs)/expenses/index.tsx:1]: Fixed

**Review by andreasalstrup — APPROVED** [review]
LGTM

**Pull request merged**